Task1/server.js

```diff
@@ -0,0 +1,22 @@
+import express from "express";
+const app = express();
+const PORT = 3000;
+const URL = "/api/greetings";
+
+//http://localhost:3000/api/greetings?Name=Egor
+
+app.get(URL, function (req, res) {
+  try {
+    const name = req.query.Name;
```

Comment:
Why did you use `Name` instead of `name`?
Remember this case as getting Query param (`req.query.name`)!
But let's try to describe logic for getting also URL param, it means `req.params.name`